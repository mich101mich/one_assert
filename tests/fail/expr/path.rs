fn main() {
    one_assert::assert!(std::vec);
    one_assert::assert!(std::vec::Vec);
    one_assert::assert!(std::f32::consts::PI);
}
