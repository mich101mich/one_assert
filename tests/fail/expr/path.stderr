error[E0423]: expected value, found module `std::vec`
 --> tests/fail/expr/path.rs:2:25
  |
2 |     one_assert::assert!(std::vec);
  |                         ^^^^^^^^ not a value

error[E0423]: expected value, found struct `std::vec::Vec`
 --> tests/fail/expr/path.rs:3:25
  |
3 |     one_assert::assert!(std::vec::Vec);
  |                         ^^^^^^^^^^^^^
  |
 ::: $RUST/alloc/src/vec/mod.rs
  |
  | pub struct Vec<T, #[unstable(feature = "allocator_api", issue = "32838")] A: Allocator = Global> {
  | ------------------------------------------------------------------------------------------------ `std::vec::Vec` defined here

error[E0308]: mismatched types
 --> tests/fail/expr/path.rs:4:25
  |
4 |     one_assert::assert!(std::f32::consts::PI);
  |                         ^^^^^^^^^^^^^^^^^^^^ expected `bool`, found `f32`
